YAPA/App.config

```diff
@@ -1,102 +1,89 @@
-            <setting name="WorkTrayIconColor" serializeAs="String">
-                <value>#00802b</value>
-            </setting>
-            <setting name="WorkMusic" serializeAs="String">
-                <value />
-            </setting>
-            <setting name="BreakMusic" serializeAs="String">
-                <value />
-            </setting>
-        </YAPA.Properties.Settings>
-    </userSettings>
+  </configSections>
+  <startup>
+    <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.6.1" />
+  </startup>
+  <userSettings>
+    <YAPA.Properties.Settings>
+      <setting name="WindowLeft" serializeAs="String">
+        <value>0</value>
+      </setting>
+      <setting name="WindowTop" serializeAs="String">
+        <value>0</value>
+      </setting>
+      <setting name="IsFirstRun" serializeAs="String">
+        <value>True</value>
+      </setting>
+      <setting name="PeriodWork" serializeAs="String">
+        <value>25</value>
+      </setting>
+      <setting name="PeriodShortBreak" serializeAs="String">
+        <value>5</value>
+      </setting>
+      <setting name="PeriodLongBreak" serializeAs="String">
+        <value>15</value>
+      </setting>
+      <setting name="SoundNotification" serializeAs="String">
+        <value>True</value>
+      </setting>
+      <setting name="LightTheme" serializeAs="String">
+        <value>True</value>
+      </setting>
+      <setting name="OpacityLevel" serializeAs="String">
+        <value>0.6</value>
+      </setting>
+      <setting name="ShadowOpacityLevel" serializeAs="String">
+        <value>0.6</value>
+      </setting>
+      <setting name="TextBrush" serializeAs="String">
+        <value>White</value>
+      </setting>
+      <setting name="CurrentScreenWidth" serializeAs="String">
+        <value>0</value>
+      </setting>
+      <setting name="CurrentScreenHeight" serializeAs="String">
+        <value>0</value>
+      </setting>
+      <setting name="CountBackwards" serializeAs="String">
+        <value>False</value>
+      </setting>
+      <setting name="MinimizeToTray" serializeAs="String">
+        <value>False</value>
+      </setting>
+      <setting name="ShowInTaskbar" serializeAs="String">
+        <value>True</value>
+      </setting>
+      <setting name="BreakTrayIconColor" serializeAs="String">
+        <value>#cc3300</value>
+      </setting>
+      <setting name="WorkTrayIconColor" serializeAs="String">
+        <value>#00802b</value>
+      </setting>
+      <setting name="WorkMusic" serializeAs="String">
+        <value />
+      </setting>
+      <setting name="BreakMusic" serializeAs="String">
+        <value />
+      </setting>
+    </YAPA.Properties.Settings>
+  </userSettings>
   <entityFramework>
-    <defaultConnectionFactory type="System.Data.Entity.Infrastructure.SqlCeConnectionFactory, EntityFramework">
+    <defaultConnectionFactory type="System.Data.Entity.Infrastructure.LocalDbConnectionFactory, EntityFramework">
```

Comment:
yeah, you're right, that will  be too much trouble to change the database, it is just because I prefer localdb and you probably want to ignore this commit.